Prepared release 1.0.0 (#206)
exasol/transformers-extension: doc/changes/changes_1.0.0.md

# Transformers Extension 1.0.0, 2024-04-05

Code name: Local model loading

## Summary

> In this Release we integrated new model load functionality which means downloaded models will now be saved 

Reviewer: ```suggestion
In this release, we integrated a new model loading functionality which means downloaded models will now be saved 
```

> in the bucketFS, meaning you can now use pretrained models without connecting to the internet as 
> long as you have saved them previously. There are also documentation updates.

Reviewer: ```suggestion
in the BucketFS. This means, the Prediction UDFs do not connect to the internet to look for model updates. There are also documentation updates.
```

### Breaking API changes

> The change in the model loading functionality means the API of calling a model has changed. 

Reviewer: ```suggestion
The change in the model loading functionality means the API for the Prediction UDFs has changed. 
```
